### PR TITLE
Awesome books: plain JavaScript with objects

### PR DESCRIPTION
**Pull request by Umuhan**
### In this pull request we made the following changes:

- [x] Create  an array of objects 

- [x] Create a function to add a new book 

- [x] Create a function to remove a book 

- [x] Display all books saved

- [x] Add new book is added to the collection.

- [x] store in the localStorage

Use of plain JavaScript.

### CODE REVIEW

**Review by Ibizugbe — COMMENTED** [review]
Hi @Umuhan 👋🏼👋🏼👋🏼, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have contributions from your learning partner. Kindly reach out to your learning partner and work with Him as this is a pair programming activity.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: CSS/styles.css [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  margin: 0 auto;
+  text-align: center;
+  font-size: 2.5rem;
+}
+
+/* .all-books {
+  margin-top: 2rem;
+  width: 50vw;
+  border: 1px solid #000;
+} */
```
Comment (Ibizugbe) [inline CSS/styles.css]: Kindly take away every commented block of code in your codebase as it makes it messy and gives it a less professional look👍🏼.

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Team👋🏼👋🏼👋🏼, 

require_changes


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] Linters are passing👍🏼.
- [x] Add, remove and display functions works properly👍🏼.
- [x] Data is preserved in local storage👍🏼.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


### NOTE: **I noticed you have gone ahead with the tasks for the future milestones in this branch, kindly note that every milestone will require a new branch and a fresh PR for it**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @TEAM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- No linter errors ✔️ 
- Correct GitHub Flow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**